hw_django/task2app/admin.py

```diff
@@ -1,3 +1,19 @@
 from django.contrib import admin
+from .models import Client, Good, Order
+
+
```

Comment:
Можно регистрировать так
```suggestion
@admin.register(ClientAdmin)
```
тогда 17 строка будет не нужна